packages/jest/src/generators/configuration/files/common/tsconfig.spec.json__tmpl__

```diff
@@ -9,14 +9,7 @@
   "files": ["src/test-setup.ts"],<% } %>
   "include": [
     "jest.config.ts",
-    "src/**/*.test.ts",
-    "src/**/*.spec.ts",<% if (supportTsx) { %>
-    "src/**/*.test.tsx",
-    "src/**/*.spec.tsx",
-    "src/**/*.test.js",
-    "src/**/*.spec.js",
-    "src/**/*.test.jsx",
-    "src/**/*.spec.jsx",<% } %>
-    "src/**/*.d.ts"
+    "{src,test}/**/*.{test,spec}.{ts<% if (supportTsx) { %>,tsx,js,jsx<% } %>}",
+    "{src,test}/**/*.d.ts"
```

Comment:
TypeScript doesn't support brace expansion here. A config like this would result in no files being included in the TS program.